docs/source-pytorch/visualize/supported_exp_managers.rst

```diff
@@ -8,8 +8,7 @@ To use `Comet.ml <https://www.comet.ml/site/>`_ first install the comet package:
 
 Configure the logger and pass it to the :class:`~lightning.pytorch.trainer.trainer.Trainer`:
 
-.. testcode::
```

Comment:
let's keep it, and I ll add installation for loggers in docs

Reply:
I can't keep it, the mock of the logger import will make the below example fail. I don't think we should block this PR for the decision of installing the loggers. This is a minor detail and here we're just looking at one of many code blocks.